### ISSUE TITLE
Subscribe checkbox shows to logged out users but doesn't subscribe them

### ISSUE THREAD

**Issue by brashrebel** [dragonslayerindustries/learndash-mailchimp#26]
If I am logged out and visit a course page which is showing the checkbox to subscribe, then check the box, then click on the Take this Course button, I am prompted to register or login. If I do, I go back to the course page but I have not been subscribed.

We either need to make it so this user _does_ get subscribed but not until they've logged in OR just don't show the subscribe option unless they are logged in already. I like the first option better.

**Comment by objectsweb** [comment]
In progress.

**Comment by objectsweb** [comment]
- Now the logged in users only can see and subscribe for any course

**Comment by brashrebel** [comment]
I realize I now have conflicting input. In another issue I stated that the subscribe option should show for logged out users but in this I said it would be ok to just hide it. I prefer showing it with an email field but since I misspoke here I guess we can just call it satisfied. Probably in a later release we will change it to show with an email field for logged out users.
